public/scripts/slash-commands.js

```diff
@@ -2270,7 +2270,7 @@ export function initDefaultSlashCommands() {
         namedArgumentList: [
             SlashCommandNamedArgument.fromProps({
                 name: 'id',
-                description: 'injection ID or variable name pointing to ID',
+                description: 'injection ID',
                 typeList: [ARGUMENT_TYPE.STRING],
                 isRequired: true,
```

Comment:
It's not required anymore, isn't it?
So can be set to false, otherwise autocomplete of stscript will still kind of force it.